application/di/init.go

```diff
@@ -138,6 +146,19 @@ func initInfrastructure() {
 	)
 }
 
+func initNetworkAccessHeaders(n networking.NetworkAccess, c *config.Config) {
+	n.AddHeaderField("User-Agent", fmt.Sprintf("%s/%s (%s;%s) %s/%s (%s;%s)",
+		"snyk-ls",
+		config.Version,
+		runtime.GOOS,
+		runtime.GOARCH,
+		c.IntegrationName(),
+		c.IntegrationVersion(),
+		"language-server",
+		config.Version,
```

Comment:
Why is config.Version used twice?

Reply:
It was to follow the `User-Agent` syntax when applied to LS, the pattern would be:
`User-Agent: snyk-cli/1.1172.0 ($GOOS;$GOARCH[;$BINARY_NAME]) [$SNYK_INTEGRATION_NAME/$SNYK_INTEGRATION_VERSION [($SNYK_INTEGRATION_ENVIRONMENT/$SNYK_INTEGRATION_ENVIRONMENT_VERSION)]]`

Which would look like:
`User-Agent: snyk-ls/20230508.144458 (DARWIN;ARM64) VS_CODE/1.20.1 (language-server/20230508.144458)`